src/bundles/analytics.js

```diff
@@ -158,7 +158,7 @@ const selectors = {
   /**
    * @param {State} state
    */
-  selectAnalyticsOptedOutPriorToDefaultOptIn: (state) => state.analytics.consent.length === 0 && state.analytics.lastDisabledAt > state.analytics.lastEnabledAt,
+  selectAnalyticsOptedOutPriorToDefaultOptIn: (state) => !state.analytics.optedOut && state.analytics.consent.length === 0 && state.analytics.lastDisabledAt > state.analytics.lastEnabledAt,
```

Comment:
this was the only change to code I was expecting. we should revert the changes to lastEnabledAt and lastDisabledAt